cpp/src/io/parquet/page_enc.cu

```diff
@@ -3219,13 +3219,13 @@ __device__ int32_t calculate_boundary_order(statistics_chunk const* s,
                                             ConvertedType ctype,
                                             uint32_t num_pages)
 {
-  if (not is_comparable(ptype, ctype)) { return BoundaryOrder::UNORDERED; }
+  if (not is_comparable(ptype, ctype)) { return static_cast<int32_t>(BoundaryOrder::UNORDERED); }
```

Comment:
should we return `BoundaryOrder` and cast at the (single) call site?

Reply:
Done!